src/sick_mrs1000_communication.cpp

```diff
@@ -42,25 +42,41 @@ SickMrs1000Communication::SickMrs1000Communication(const std::string &hostname,
                                                    ScanAndCloudParser* parser)
 : SickTimCommonTcp(hostname, port, timelimit, parser),
   scan_and_cloud_parser_(parser),
-  cloud_pub_(nh_.advertise<sensor_msgs::PointCloud2>("cloud", 300)),
-  diagnosed_cloud_publisher_(cloud_pub_, diagnostics_,
-    diagnostic_updater::FrequencyStatusParam(&expectedFrequency_, &expectedFrequency_, 0.1, 10),
-    diagnostic_updater::TimeStampStatusParam(-1, 1.3 * 1.0/expectedFrequency_ - config_.time_offset)
-  )
+  cloud_pub_(nh_.advertise<sensor_msgs::PointCloud2>("cloud", 300))
 {
-  expectedFrequency_ = 12.5; // by data sheet 50 Hz, 4 x 12,5 Hz
+
+  updater_mrs_1000_ = new marble::DiagnosticUpdater("/"+namespace_+"/"+"cloud", nh_);
```

Comment:
I believe the generic diagnostic from the `SickTimCommon` class also be added to this updater. Otherwise a user could declare a dependency on this `cloud` diagnostic, but wouldn't get any of the errors from the base class propagated up.

Reply:
I did not want a debug topic to publish generics diagnostics. It seemed moot to have the same diagnostic message flashing twice.

Reply:
Done